client/src/app/site/pages/meetings/pages/projectors/modules/projector-detail/components/projector-detail/projector-detail.component.ts

```diff
@@ -109,6 +109,18 @@ export class ProjectorDetailComponent extends BaseMeetingComponent implements On
         return Math.floor(this.projector.height / 100);
     }
 
+    /** stores the decimal places that could not be scrolled
+     *  at once for the projector height */
+    public get summedOverflowSteps(): number {
+        return this._summedOverflowSteps;
+    }
+
+    public set summedOverflowSteps(value: number) {
+        this._summedOverflowSteps = value;
+    }
```

Comment:
No real need to write getters, if you're only going to use the property class-internally. Just read the private property as-is.